### DISCUSSION TITLE
not found the page static files

### DISCUSSION THREAD

**Discussion by Gitmpb** [coder/code-server#6486] [General]
![2023-10-12_163310](https://github.com/coder/code-server/assets/16250757/080be45b-8b2c-4c45-ba54-98c99944deac)

**Comment by code-asher** [comment]
Try a trailing slash `/proxy/8081/`  You can also try `/absproxy/8081/`.  If those both fail then most likely your app needs to be configured to understand it is being hosted behind a base path.
